chstrings/__init__.py

```diff
@@ -21,6 +21,12 @@ def _preprocess_variables(config, strings):
     else:
         strings['lead_section_hint'] = ''
 
+    reliable_sources_hint_link = _link(
+        config.reliable_sources_link,
+        config.reliable_sources_link_title)
+    strings['reliable_sources_hint'] = \
+        flask.Markup(strings['reliable_sources_hint']) % reliable_sources_hint_link
```

Comment:
Nope, that was my mistake. I don't have an active version of this running but was hoping to save you some time in implementation—is this a simple fix that you can test? Or should I scrap the PR?